website/docs/components/dropdown/partials/code/component-api.md

```diff
@@ -17,6 +17,45 @@ The Dropdown component is composed of different child components each with their
 ### Dropdown
 
 <Doc::ComponentApi as |C|>
+  <C.Property @name="<[DD].ToggleButton>" @type="yielded component">
```

Comment:
Following our naming convention, should this be just `[D]`?

Reply:
@alex-ju Yes, but unfortunately we documented it (in the how to use) as `<Hds::Dropdown as |dd|>` and now this `dd` [is all over the places](https://github.com/search?q=%22%3CHds%3A%3ADropdown+as%22++user%3Ahashicorp&type=code&ref=advsearch) :( 
So I tried to keep the naming in the "Component API" block consistent with what we have in the "how to use" block. More than happy to update the "how to use" block above, in this PR, if that's OK for you.

Reply:
ah, I see.. yeah, that's a good point. I guess consistent within the component is more of a priority. If not an overkill, I would like to make this change, but can do it in a follow up if you'd like to get this in sooner

Reply:
No, easy/cleaner to do in this context

Reply:
@alex-ju on a second thought, there were too many places where they should have been updated, for consistency, so I created a new PR: https://github.com/hashicorp/design-system/pull/2247